app/views/endemics/species-numbers-exception.njk

```diff
@@ -0,0 +1,47 @@
+{% extends './layouts/layout.njk' %}
+
+{% block pageTitle %}You cannot continue with your claim - {{ serviceName }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: backLink,
+    attributes: {
+      id: "back"
+    }
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">You cannot continue with your claim</h1>
+      <br>
+      <p class="govuk-body">
+      You do not have the <a href="https://apply-for-an-annual-health-and-welfare-review.defra.gov.uk/apply/guidance-for-farmers">minimum number of livestock</a>.
+    </p>
+      <p class="govuk-body">If you selected the wrong option, you’ll need to change your answer.</p>
+      <p class="govuk-body">
+        <a class="govuk-link" href={{ endemicsWhichReviewAnnualPageUrl }}>Change your answer for number of livestock</a>
+      </p>
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+           Your claim will be checked by our team.
+        </strong>
+      </div>
+            <h2 class="govuk-heading-m">Get help with your claim</h2>
+      <p class="govuk-body">If you need help with your application, contact the Rural Payments Agency.</p>
```

Comment:
Check with Alessia if this should say 'claim' not 'application'. I asked about the same thing on a different error page and they asked me to change it